lib/models/blueprint.js

```diff
@@ -1536,6 +1519,12 @@ Blueprint.lookup = function (name, options) {
     }
   }
 
+  // Check if `name` itself is a path to a blueprint:
+  let blueprintPath = path.resolve(name);
```

Comment:
What happens if someone accidentally tries, for example, `ember g tests foo`? I believe right now they would get the unknown blueprint error, but after this change it would attempt to load it as a blueprint and throw in unknown ways.

Reply:
Good call, I guess we still need a `/` check?

Reply:
Should be fixed now.